elixir/apps/domain/lib/domain/relay_token.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Domain.RelayToken do
+  use Ecto.Schema
+
+  @primary_key false
+  @foreign_key_type :binary_id
+  @timestamps_opts [type: :utc_datetime_usec]
+
+  schema "relay_tokens" do
+    field :id, :binary_id, primary_key: true, autogenerate: true
+
+    field :secret_hash, :string, redact: true
+    field :secret_salt, :string, redact: true
```

Comment:
This likely isn't the time to change this, but we probably want to be using something like Argon2 rather than keeping track of the salt ourselves.